### PR TITLE
Add downstream RKE1 Linode clusters support

### PR DESCRIPTION
**Pull request by git-ival**
This PR splits the downstream `clusters-rke1` module into two separate cloud provider-specific modules (aws and linode).
Included are some minor fixes to the `rancher-examples/rancher-setup` example modules

### CODE REVIEW

File: clusters-rke1/README.md [hunk 1]
```diff
@@ -1,96 +1,42 @@
...
-| <a name="input_volume_type"></a> [volume\_type](#input\_volume\_type) | Type of storage volume to use | `string` | `"gp2"` | no |
-
-## Outputs
-
-| Name | Description |
-|------|-------------|
-| <a name="output_create_node_reqs"></a> [create\_node\_reqs](#output\_create\_node\_reqs) | n/a |
-| <a name="output_node_template"></a> [node\_template](#output\_node\_template) | n/a |
-<!-- END OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
+This terraform is for creating the downstream RKE clusters to test Ranchers scaling ability. It contains two root modules, one for AWS clusters and one for Linode clusters.
```
Comment (floatingman) [inline clusters-rke1/README.md:1]: Maybe there needs to be instructions for the provisioning on AWS also?

**Review by floatingman — COMMENTED** [review]
Just one comment to keep the instructions for AWS based clusters.

File: clusters-rke1/README.md (hunk 1 above)
Comment (git-ival) [reply clusters-rke1/README.md:1]: This is a generic README that applies to both modules, the specific examples provided just happen to be for the linode module. I think its ok as-is for the current PR. The required `terraform.tfvars` for the AWS module can be mostly copied from the README with some minor changes based on the variables that exist or don't exist in the AWS module.

**Review by floatingman — APPROVED** [review]
LGTM

**Pull request merged**